.github/scripts/simpleauth-azdo-pipelines.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+x=1
+restUrl="https://dev.azure.com/mseng/VSIoT/_apis/build/latest/$2?api-version=6.0-preview.1"
```

Comment:
is there non-preview api available?

Reply:
copy this api from https://docs.microsoft.com/en-us/rest/api/azure/devops/pipelines/runs/run-pipeline?view=azure-devops-rest-6.0